### ISSUE TITLE
Call to undefined method SetAppKey::getLaravel() on install

### ISSUE THREAD

**Issue by daveismyname** [AsgardCms/Platform#432]
### Summary of issue
I'm running the cli installer after the application url is set an error is displayed. 

### Steps to reproduce
The error is: 

```
  [Symfony\Component\Debug\Exception\FatalThrowableError]
  Call to undefined method Modules\Core\Console\Installers\Scripts\SetAppKey:
  :getLaravel()
```

### System Details

- Operating System
Mac Os High Sierra

- PHP Version
PHP 7.1.4

- Laravel Version
Fresh install so I assume 5.5

- `Asgardcms/Platform` version
3.3.0

**Comment by joshbrw** [comment]
Can't recreate this with the latest version of the Asgard installer.

**Comment by joshbrw** [comment]
So it looks like this happens when the Application is set to Production environment and you try and install. Could you try it with it set to Development mode?

**Comment by daveismyname** [comment]
I've just tried installing the installer and installing a fresh one from scratch it's installed this time with no errors.

**Comment by joshbrw** [comment]
@daveismyname thanks for confirming. This is actually an edge-case bug which I've fixed in #433
